src/pybind/mgr/cephadm/module.py

```diff
@@ -90,6 +90,9 @@ def remoto_has_connection(self):
 
 CEPH_TYPES = set(CEPH_UPGRADE_ORDER)
 
+# Location of the cephadm binary in each host
+CEPHADM_TOOL_PATH = '/var/lib/ceph/%s/cephadm/%s' # Variable parts are <cluster fsid> and <mgr id>
```

Comment:
Same here, constants should be _constant_

Reply:
changed.